Updating the docs for tls.enabled (#3493)
knative/docs: docs/eventing/samples/kafka/channel/index.md

*NOTE:* It is important to use the same keys; `user`, `password` and `saslType`.

> ### SASL authentication with public CA certs
> 
> If you are using SASL against public CA certs, use the `tls.enabled=true` flag, rather than the `ca.crt` argument when creating the secret:

Reviewer: ```suggestion
### SASL authentication using public CA certificates

If you want to use SASL with public CA certificates, you must use the `tls.enabled=true` flag, rather than the `ca.crt` argument, when creating the secret. For example:

```
